pkg/localmetrics/localmetrics.go

```diff
@@ -134,6 +139,11 @@ func NewMetricsCollector(store cache.Cache) *MetricsCollector {
 			ConstLabels: prometheus.Labels{"name": operatorName},
 			Buckets:     []float64{0.001, 0.01, 0.1, 1, 10, 30, 60, 120, 240},
 		}, []string{"controller"}),
+
+		apiCallCount: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Name: "aws_account_operator_api_requests_total",
+			Help: "Counter incremented for each API request.",
+		}, []string{"controller", "method", "resource", "status"}),
```

Comment:
Please add the const label `ConstLabels: prometheus.Labels{"name": operatorName},`

Reply:
Ah, thanks, will do.

Reply:
Done.